[2025-06-27T04:30:48.683+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:30:48.738+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-06-04T05:00:00+00:00 [queued]>
[2025-06-27T04:30:48.754+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-06-04T05:00:00+00:00 [queued]>
[2025-06-27T04:30:48.756+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:30:48.793+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-06-04 05:00:00+00:00
[2025-06-27T04:30:48.817+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3021) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:30:48.828+0000] {standard_task_runner.py:72} INFO - Started process 3052 to run task
[2025-06-27T04:30:48.823+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0', 'start_task', 'scheduled__2025-06-04T05:00:00+00:00', '--job-id', '750', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmp0wxn_sf5']
[2025-06-27T04:30:48.835+0000] {standard_task_runner.py:105} INFO - Job 750: Subtask start_task
[2025-06-27T04:30:49.001+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0.start_task scheduled__2025-06-04T05:00:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:30:49.173+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-04T05:00:00+00:00'
[2025-06-27T04:30:49.186+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:30:49.191+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T04:30:49.198+0000] {logging_mixin.py:190} INFO - ds: 2025-06-04
[2025-06-27T04:30:49.199+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T04:30:49.242+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:30:49.263+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0, task_id=start_task, run_id=scheduled__2025-06-04T05:00:00+00:00, execution_date=20250604T050000, start_date=20250627T043048, end_date=20250627T043049
[2025-06-27T04:30:49.477+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:30:49.751+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:30:49.760+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
